paper-styles.d.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ *   https://github.com/Polymer/gen-typescript-declarations
+ *
+ * To modify these typings, edit the source file(s):
+ *   paper-styles.html
+ */
+
+/// <reference path="../iron-flex-layout/iron-flex-layout.d.ts" />
+/// <reference path="color.d.ts" />
+/// <reference path="default-theme.d.ts" />
+/// <reference path="shadow.d.ts" />
+/// <reference path="typography.d.ts" />
+
+/**
+ * The `<paper-styles>` component provides simple ways to use Material Design CSS styles
+ * in your application. The following imports are available:
+ *
+ * 1. [color.html](https://github.com/PolymerElements/paper-styles/blob/master/color.html):
+ * a complete list of the colors defined in the Material Design [palette](https://www.google.com/design/spec/style/color.html)
+ *
+ * 2. [default-theme.html](https://github.com/PolymerElements/paper-styles/blob/master/default-theme.html): text,
+ * background and accent colors that match the default Material Design theme
+ *
+ * 3. [shadow.html](https://github.com/PolymerElements/paper-styles/blob/master/shadow.html): Material Design
+ * [elevation](https://www.google.com/design/spec/what-is-material/elevation-shadows.html) and shadow styles
+ *
+ * 4. [typography.html](https://github.com/PolymerElements/paper-styles/blob/master/typography.html):
+ * Material Design [font](http://www.google.com/design/spec/style/typography.html#typography-styles) styles and sizes
+ *
+ * 5. [demo-pages.html](https://github.com/PolymerElements/paper-styles/blob/master/demo-pages.html): generic styles
+ * used in the PolymerElements demo pages
+ *
+ * We recommend importing each of these individual files, and using the style mixins
+ * available in each ones, rather than the aggregated `paper-styles.html` as a whole.
+ */
+interface PaperStylesElement extends Polymer.Element {
```

Comment:
This element shouldn't exist. It's here because it has a `@pseudoElement` annotation in the source. I assume this is here as a hack to get paper-styles documentation into our doc viewer. Ideally we would find another way to do that, since this is definitely not an element. In the interest of time, though, I'm going to add a way to configure identifiers that should be omitted and add it to this repo's `gen-tsd.json` file.